core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.ts

```diff
@@ -48,6 +48,7 @@ require('services/context.service.ts');
 require('services/editability.service.ts');
 require('services/id-generation.service.ts');
 require('services/user.service.ts');
+require('third-party-imports/wave-surfer.scripts.ts');
```

Comment:
rename this to `wave-surfer.import.ts`

Reply:
also, does it make sense to load the variable into `window.WaveSurfer`? Can we just load it to a local variable named `WaveSurfer` here and use it?

Reply:
@vojtechjelinek I think in my GSoC proposal we agreed to follow *scripts.ts naming?

Reply:
I think that the final conclusion was to use `*.import.ts` for all the files that only contain requires/imports. I may have made a mistake in some of my comments. But I think that the import naming is much more decriptive.

Reply:
@vojtechjelinek Renamed the file. But had a doubt about your second comment.

Are you saying to modify the `wave-surfer.import.ts` to something like
```javascript
module.exports = require(
  'static/wave-surfer-js-2.2.1/wavesurfer.min.js');
```

and modifying the require statement in `audio-translation-bar.directive.ts` to something like
```javascript
var WaveSurfer = require('third-party-imports/wave-surfer.import.ts');
```
and use that in that file?

In my opinion, I think it looks weird we don't assign a variable to require statements in any other file. I think we should do that work in `.import.ts` file itself. What do you think?

Reply:
We actually use this for example for the hashes already https://github.com/oppia/oppia/blob/ff31fc3d4ffae61658ee9c5a14418ef2df174e48/core/templates/domain/utilities/url-interpolation.service.spec.ts#L28

from my POV it doesn't make sense to make the library available for the whole window namespace if it is required just in the one file.

Reply:
Ok, Makes sense. I'll update the code.

Reply:
@vojtechjelinek Done! Also removed the custom declarations as they are no longer needed.